chapter06/mssw/exercise1.pl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/perl
+use v5.10;
+use strict;
+use warnings;
+use Storable qw( nstore retrieve );
+use Data::Dumper;
+
+require '/home/codio/workspace/chapter06/mssw/file_reader.pm';
```

Comment:
@ardavey AFAIK `use lib` still uses paths relative to the execution location, so you'd need something like `FindBin` to get the script's location? Also, can you compile a module using `use lib`? I'd thought it was only for adding directories to the include paths.